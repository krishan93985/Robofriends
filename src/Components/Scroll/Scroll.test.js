import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import Scroll from './Scroll';

let wrapper;

beforeEach(() => {
    const mockChildrenComponent = () => <div>This is mock CardArray</div> ; 
    const mockProps = {
        children:mockChildrenComponent
    }
    wrapper = shallow(<Scroll {...mockProps} />);
})

describe('testing scroll component', () => {
    it('renders correctly', () => {
        expect(toJson(wrapper)).toMatchSnapshot();
    })
})