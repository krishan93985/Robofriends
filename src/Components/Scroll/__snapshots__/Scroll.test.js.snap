// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`testing scroll component renders correctly 1`] = `
<div
  style={
    Object {
      "border": "2px solid black",
      "height": "600px",
      "overflowY": "scroll",
    }
  }
>
  <Component />
</div>
`;
